### ISSUE TITLE
Show a better web-page for an invalid ROR IDs

### ISSUE THREAD

**Issue by paulmillar** [ror-community/ror-api#200]
If I put a ROR ID in my web-browser, I see a short summary of information about that institute; e.g., https://ror.org/01js2sh04.  This is good.

If I put an incorrect ID into my web-browser, then I see only a blank page; e.g,. https://ror.org/0-no-such-ID.  This is bad.

It would be better if the page for an invalid ROR ID included some clear indication to the user that the ID was invalid.  The page might also provide some links that could allow a user to recover from this situation (e.g,. link to the ROR main page, link to the search facility, etc).

On a technical level, the server should probably return a `404` status code.  This would allow automated scripts to understand the nature of the problem.  Currently the server returns a status code of  `200`, which indicates that there was no problem.

**Comment by paulmillar** [comment]
Note that the work to resolve issue #99 may be related.

**Comment by lizkrznarich** [comment]
@paulmillar You're right that the website UI should show a 404, however, the API does (where the majority of ROR usage happens) correctly returns a 404. For automated scripts, we would certainly recommend make requests to the API directly, ex https://api.ror.org/organizations/ror.org/0-no-such-ID, as the website is not intended to sustain heavy programmatic usage. Moving this issue over to the ror-site repo, as it's a UI rather than API issue.
